pages/partners/index.tsx

```diff
@@ -46,6 +46,24 @@ const partnersData: PartnersListingProps[] = [
     imageHeight: '362px',
     url: 'https://www.lgbtqseniorhousing.org/',
   },
+  {
+    name: 'The Equality Fund',
+    imageSrc: '/ef.jpg',
+    imageHeight: '58px',
```

Comment:
how are you choosing these heights?

Reply:
Varun made the width a constant 360px so the height is just scaled down to match the original dimensions of the logo